Dockerfile

```
@@ -1,40 +1,18 @@
-FROM ubuntu:20.04
+FROM python:3.11.9
```

```suggestion
FROM python:3.11.9-slim
```
Using `python:3.11.9` increases the total size of the image. `python:3.11.9-slim` looks good enough